README.md

```diff
@@ -76,6 +76,7 @@ config.providers = [
   {
     provider: :rollbar,
     api_key: ENV['ROLLBAR_KEY'],
+    enabled: Rails.env.production?
```

Comment:
I dunno, I like it on by default in staging. If you get a 500 in staging, it's a pain to grep through the logs to find out what the actual error was. Though `['production', 'staging'].include?(Rails.env)` might be more appropriate here to exclude `test`